src/services/albums-service.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+const ALBUMS_API = 'http://localhost:8080/albums';
+
+// export const createAlbum = async (album) => {
```

Comment:
Do we still need this here?

Reply:
Oh oops yeah now that the liked albums gives back the album data I can add that instead

Reply:
Fixed liked albums to not use this API and removed this file